### ISSUE TITLE
No error message with an invalid out language

### ISSUE THREAD

**Issue by abelrgr** [KhaledMohamedP/translate-json-object#4]
When a non existent language is used, there is no error message:
```javascript
TJO.translate(example, 'ch')
    .then(function(data) {
      console.log(data);
    }).catch(function(err) {
      console.log('error ', err)
    });
```

Instead of that there is a error message in the console and the node closes:

```
/test/expressjs/node_modules/translate-json-object/lib/service/yandex.js:29
				var result = res.text[0].split(hashSimple);
				                     ^

TypeError: Cannot read property '0' of undefined
    at /test/expressjs/node_modules/translate-json-object/lib/service/yandex.js:29:26
    at Request.handler [as _callback] (/test/expressjs/node_modules/yandex-translate/index.js:14:5)
    at Request.self.callback (/test/expressjs/node_modules/yandex-translate/node_modules/request/request.js:199:22)
    at emitTwo (events.js:106:13)
    at Request.emit (events.js:194:7)
    at Request.<anonymous> (/test/expressjs/node_modules/yandex-translate/node_modules/request/request.js:1036:10)
    at emitOne (events.js:101:20)
    at Request.emit (events.js:191:7)
    at IncomingMessage.<anonymous> (/test/expressjs/node_modules/yandex-translate/node_modules/request/request.js:963:12)
    at emitNone (events.js:91:20)
```
BTW: The lib save me a ton of work ;)

**Comment by KhaledMohamedP** [comment]
@0bot this seems to be a problem with [yandex-translate](https://www.npmjs.com/package/yandex-translate) module. I am planing to actually deprecate this module since it hasn't been updated lately. 


Let me know if this is an urgent issue at the moment?

**Comment by KhaledMohamedP** [comment]
This is now implemented in v2.3 in yandex & google
